src/Engines/MeiliSearchEngine.php

```diff
@@ -62,7 +62,11 @@ public function update($models)
                 return;
             }
 
-            return array_merge($searchableData, $model->scoutMetadata());
+            return array_merge(
+                [$model->getKeyName() => $model->getScoutKey()],
```

Comment:
`$model->getKeyName()` because in line 73 we set the primary key to `$models->first()->getKeyName()`